### PR TITLE
Added three methods to UIView+Constraints category

### PR DESCRIPTION
**Pull request by nservidio**

### CODE REVIEW

File: src/UIView/UIView+Constraints.h [hunk 1]
```diff
@@ -161,6 +161,30 @@
...
+- (void)constrainView:(UIView *)view toBottomOfView:(UIView *)positioningView;
+
+/**
+ *  Constrain the width of two views to be equal
+ *
+ *  @param view       the view to constrain
+ *  @param sizingView the view to use as a width reference
+ */
+- (void)constrainView:(UIView *)view toWidthOfView:(UIView *)sizingView;
+
```
Comment (Benuuu) [inline src/UIView/UIView+Constraints.h:187]: Can you also add the height one while you're here?

**Comment by Benuuu** [comment]
:shipit:

**Comment by brightredchilli** [comment]
:ship:

**Comment by loganwright** [comment]
Did anyone update the podspec for this on our specs repo? @nservidio

**Comment by brightredchilli** [comment]
ill add one tomorrow. i need to refresh my memory on this.

**Comment by loganwright** [comment]
@brightredchilli Nick and I already updated it today :)

**Comment by brightredchilli** [comment]
:+1:

**Pull request merged**